pkg/apis/extensions/v1beta1/types.go

```diff
@@ -36,6 +36,14 @@ type ScaleStatus struct {
 
 	// label query over pods that should match the replicas count. More info: http://releases.k8s.io/HEAD/docs/user-guide/labels.md#label-selectors
 	Selector map[string]string `json:"selector,omitempty"`
+
+	// label selector for pods that should match the replicas count. This is a serializated
+	// version of both map-based and more expressive set-based selectors. This is done to
+	// avoid introspection in the clients. The string will be in the same format as the
+	// query-param syntax. If the target type only supports map-based selectors, both this
+	// field and map-based selector field are populated.
+	// More info: http://releases.k8s.io/HEAD/docs/user-guide/labels.md#label-selectors
+	TargetSelector string `json:"targetSelector,omitempty"`
```

Comment:
@bgrant0607 for 1.2, is there a reason we don't use the same external representation as extensions.v1beta1 ReplicaSet and Deployment? If we want to move to a string externally, shouldn't that be done for all three of those objects at once in the same api version? If we used the same representation as ReplicaSet/Deployment, all the conversions would already exist, and populating the existing `Selector map[string]string` would be trivial as well